pandas/core/arrays/categorical.py

```diff
@@ -2412,6 +2412,9 @@ def isin(self, values):
         code_values = code_values[null_mask | (code_values >= 0)]
         return algorithms.isin(self.codes, code_values)
 
+    def replace_list(self, to_replace, value, inplace: bool = False):
+        return self.replace(to_replace, value, inplace)
+
```

Comment:
Where is this used?

Reply:
sorry, this and the change below it were experiments in addressing the dict case and list-list case for categoricals, which turned out to be more complex than expected and are not intended to be in this pr